executor/predictor/predictor_process.go

```diff
@@ -100,6 +100,11 @@ func (p *PredictorProcess) transformInput(node *v1.PredictiveUnit, msg payload.S
 	modelName := p.getModelName(node)
 
 	if callModel || callTransformInput {
+		msg, err := p.Client.Chain(p.Ctx, modelName, msg)
```

Comment:
This makes sense, also for the Tensorflow protocol - thinking it may be worth also implementing this for the `transformOutput` method?

Reply:
Here if the chain function fails and throws an error, the message will not be logged, is that a concern?

Reply:
Good spot @axsaucedo , just added the same for `transformOutput`.

@SachinVarghese that's a good point. Although if the chaining fails, the request will also fail. Should we log the request if we know it's gonna fail?

Reply:
I had a look at this again, current behavior seems to be good.